.github/workflows/release.yml

```diff
@@ -0,0 +1,39 @@
+name: Python package build and publish
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
```

Comment:
Is it possible to test this job without making a release?

Reply:
We could trigger it on push and tell it to use the credentials for the "test PyPI" instance rather than the production one.

I'm not really sure it's necessary though, we can check everything up to point of the upload pretty easily.  https://github.com/rpm-software-management/createrepo_c/pull/256/checks?check_run_id=2099124911

Reply:
https://github.com/rpm-software-management/libcomps/runs/2100839496?check_suite_focus=true